### PR TITLE
Fixing Issues: Validation of tenant in ListTiers #1311

### PR DESCRIPTION
**Pull request by PravinRanjan10**
This is fix for: #1311 

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:
1. 
SSPcoming

2. 
BucketCreate-Kal

**Special notes for your reviewer**:

### CODE REVIEW

File: api/pkg/backend/service.go [hunk 1]
```diff
@@ -910,15 +910,16 @@ func (s *APIService) ListTiers(request *restful.Request, response *restful.Respo
...
-	//Validation of tenantId of tier:
-	actx := request.Attribute(c.KContext).(*c.Context)
-	token := actx.AuthToken
-	if !ValidateTenant(token, tenantId) {
-		errMsg := fmt.Sprintf("tenantId:%v in request is invalid:", tenantId)
-		log.Error(errMsg)
-		response.WriteError(http.StatusBadRequest, errors.New(errMsg))
-		return
-	}
+	// TODO: FIX ME
```
Comment (anvithks) [inline api/pkg/backend/service.go:913]: Can you please write the details for the FIXME.

**Review by anvithks — CHANGES_REQUESTED** [review]
Please add a detailed comment for the TODO.
Can the commented code be removed?

File: api/pkg/backend/service.go (hunk 1 above)
Comment (PravinRanjan10) [reply api/pkg/backend/service.go:913]: Done

**Review by anvithks — APPROVED** [review]
LGTM

**Review by NajmudheenCT — APPROVED** [review]
LGTM

**Pull request merged**